enaml/scintilla/scintilla.py

```diff
@@ -8,11 +8,13 @@
 import uuid
 
 from atom.api import (
-    Atom, Int, Constant, Enum, Event, Typed, ForwardTyped, observe, set_default
+    Atom, Int, Constant, Enum, Event, Typed, List, ForwardTyped, Tuple,
+    Unicode, observe, set_default
 )
-
+from enaml.image import Image
 from enaml.core.declarative import d_
 from enaml.widgets.control import Control, ProxyControl
+from future.builtins import str
```

Comment:
Please keep external (external to enaml) grouped before enaml imports it helps identifying the dependencies faster.